test/scales/tickGeneratorsTests.ts

```diff
@@ -1,72 +1,84 @@
 ///<reference path="../testReference.ts" />
 
-describe("Tick generators", () => {
-  describe("interval", () => {
-    it("generate ticks within domain", () => {
-      let start = 0.5, end = 4.01, interval = 1;
-      let scale = new Plottable.Scales.Linear();
-      scale.domain([start, end]);
-      let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
-      assert.deepEqual(ticks, [0.5, 1, 2, 3, 4, 4.01], "generated ticks contains all possible ticks within range");
-    });
+describe("Scales", () => {
+  describe("Tick generators", () => {
+    describe("Interval", () => {
+      let scale: Plottable.Scales.Linear;
 
-    it("domain crossing 0", () => {
-      let start = -1.5, end = 1, interval = 0.5;
-      let scale = new Plottable.Scales.Linear();
-      scale.domain([start, end]);
-      let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
-      assert.deepEqual(ticks, [-1.5, -1, -0.5, 0, 0.5, 1], "generated all number divisible by 0.5 in domain");
-    });
+      beforeEach(() => {
+        scale = new Plottable.Scales.Linear();
+      });
 
-    it("generate ticks with reversed domain", () => {
-      let start = -2.2, end = -7.6, interval = 2.5;
-      let scale = new Plottable.Scales.Linear();
-      scale.domain([start, end]);
-      let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
-      assert.deepEqual(ticks, [-7.6, -7.5, -5, -2.5, -2.2], "generated all ticks between lower and higher value");
-    });
+      it("generates ticks within domain", () => {
+        let start = 0.5, end = 4.01, interval = 1;
+        scale.domain([start, end]);
+        let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
 
-    it("passing big interval", () => {
-      let start = 0.5, end = 10.01, interval = 11;
-      let scale = new Plottable.Scales.Linear();
-      scale.domain([start, end]);
-      let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
-      assert.deepEqual(ticks, [0.5, 10.01], "no middle ticks were added");
-    });
+        assert.strictEqual(ticks.length, 6, "ticks are generated");
+        ticks.forEach((tick) => {
+          assert.operator(start, "<=", tick, "tick " + tick + " should be greater than the lower bound of the domain");
+          assert.operator(tick, "<=", end, "tick " + tick + " should be less than the upper bound of the domain");
```

Comment:
Optionally, you can use a template string here, but I think the tick value doesn't even need to be in the assert message, since it's already passed in as one of the arguments.